lib/bee/writer/Neo4jWriter.rb

```diff
@@ -58,9 +73,9 @@ def getNode(key, value, add=false)
       mynode = Neo4j::Node.load(@nodecache[value])
       if (!mynode) # Cache miss
         mynodes = Neo4j::Label.find_nodes(:node, key, value)
-        if (mynodes.size == 1) # we found it!
+        if (mynodes.count == 1) # we found it!
```

Comment:
With non-embedded DB, the size() method returns nil, which apparently means the enumerable cannot immediately calculate the result. count() seems to work.